impl/src/main/java/org/glassfish/json/MapUtil.java

```diff
@@ -18,14 +18,15 @@
 
 import org.glassfish.json.api.BufferPool;
 
-import javax.json.JsonArrayBuilder;
-import javax.json.JsonObjectBuilder;
-import javax.json.JsonValue;
 import java.math.BigDecimal;
 import java.math.BigInteger;
 import java.util.Collection;
 import java.util.Map;
 
+import javax.json.JsonArrayBuilder;
+import javax.json.JsonObjectBuilder;
+import javax.json.JsonValue;
+
```

Comment:
What is the point of this change? Is it required for everyone to update existing imports to fit the setup of his IDE or should the one doing a change update his/her IDE to follow conventions used by the project? I have nothing against changing wildcard imports to fqn ones but I don't like every other change to come up with re-sorted imports

Reply:
That was accidental. Sorry about that.

Reply:
OK, imports are reverted back to their previous state. Sorry that it took me so long to get to this after complaining about you taking so long, but I actually gave up and switched to a different implementation.

Also, I couldn't find a style guide or coding conventions anywhere. I should have caught the change in imports, but it's hard to set my IDE prefs if I don't know what they should be set to.